linkcovery/cli/__init__.py

```
@@ -39,7 +85,145 @@ def stats() -> None:
+    data_dir = db_path.parent
+    table.add_row("Configuration", str(config_file), config_size, config_modified)
+    table.add_row("Database", str(db_path), db_size, db_modified)
+    table.add_row("Data Directory", str(data_dir), "-", "-")
+
+    console.print(table)
+
+
+@cli_app.command(rich_help_panel="Link Management")
+@handle_errors
+def mark(
+    link_ids: list[int] = typer.Argument(..., help="Link IDs to mark"),
```

Ruff (0.15.0)

[warning] Do not perform function call `typer.Argument` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)